tests/fixtures/upgrade/resources.py

```
@@ -0,0 +1,193 @@
+    admin_client,
+    worker_node1,
+    rhel_latest_os_params,
+    updated_default_storage_class_ocs_virt,
+):
+    golden_images_namespace_name = py_config["golden_images_namespace"]
+    dvs_list = []
+    artifactory_secret = get_artifactory_secret(namespace=golden_images_namespace_name)
+    artifactory_config_map = get_artifactory_config_map(namespace=golden_images_namespace_name)
+
+    for sc in py_config["storage_class_matrix"]:
+        storage_class = [*sc][0]
```

Ruff (0.15.21)

[warning] Prefer `next(iter(sc))` over single element slice

Replace with `next(iter(sc))`

(RUF015)